kodilla-stream/src/test/java/com/kodilla/exception/homework/WarehouseTestSuite.java

```diff
@@ -0,0 +1,40 @@
+package com.kodilla.exception.homework;
+
+import org.junit.Test;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.junit.Assert.*;
+
+public class WarehouseTestSuite {
+
+    @Test
+    public void testGetOrderExists() throws OrderDoesntExistException {
+        // given
+        Warehouse warehouse = new Warehouse();
+        warehouse.addOrder(new Order("5/5/2020"));
+
+        // when
+        Order result = warehouse.getOrder("5/5/2020");
+
+        // then
+        assertEquals(new Order("5/5/2020"), result);
```

Comment:
assertEquals("5/5/2020", result.getNumber())

Reply:
WoW! to w resultacie możemy metodę getNumber brać jak testuję czy jest zamówienie, nie wpadałbym na to